sage-android/Sage/app/src/main/java/blueprint/com/sage/main/fragments/AnnouncementsListFragment.java

```diff
@@ -85,11 +98,22 @@ public void initializeViews() {
 
     @Override
     public void onRefresh() {
+        makeRequest();
+    }
+
+    @Override
+    public void onResume() {
+        makeRequest();
```

Comment:
also, we shouldn't make a list request every single time we call on resume - it might result in alot of redundancy.